### DISCUSSION TITLE
Write-up of possible optimizations to be done on CF side

### DISCUSSION THREAD

**Discussion by Faupi** [HttpRafa/modflared#25] [Ideas]
As somebody that hasn't messed around with CloudFlare and similar setups _much_ before, I was wondering if there could be any optimizations possible with the free tier there, it could be cool to maybe even have a write-up in the README or such so people could get the most out of it. For example how to avoid possible timeouts (I've had a few reported, possibly due to the rotating IPs I'd assume), if there would be something to improve general performance, etc.

But great project! Pulled me out of trouble when I found that my new ISP doesn't allow public IPs haha

**Comment by HttpRafa** [comment]
The problem is that there is not much to optimize because the Cloudflare tunnels are not actually made for Minecraft. The timeout probably occurs because of the amount of time you are on the Minecraft server.

**Reply by HttpRafa** [reply]
In my experience how and when problems occur varies greatly.

**Reply by Faupi** [reply]
Would for example enabling protocol optimizations (like Brotli by any weird chance) have **any** effect on the stability or performance? Currently it feels like generally nothing on CF side has an effect, though I also haven't tested much of it with my limited knowledge about the matter. Generally anything that _could_ help would be a great tip ^^
We will be testing it a bunch with our new server - if I happen to make any discoveries I could share them, though again my expertise in this field is so bare that it might not make a dent xP

**Reply by HttpRafa** [reply]
I have already done some tests but have not found anything yet. If you find something, please let me know and I will write it in the README.
